Automattic/newspack-plugin
# feat(ras-acc): add donation cancellation email template

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://app.asana.com/0/1206810965479792/1205743341940640/f

This PR adds a new donation cancellation email:

![Screenshot 2024-04-10 at 15 40 37](https://github.com/Automattic/newspack-plugin/assets/17905991/40c78651-8f1b-4398-a5ca-3a235bb99c49)

![Screenshot 2024-04-10 at 15 39 30](https://github.com/Automattic/newspack-plugin/assets/17905991/88ef770d-cfa5-411e-b3b9-b5e063889138)

Note: ~~I've populated the cancellation email with super basic copy based mostly off of the email template for receipt. Since we are revisiting email design in ras-acc and this is going into epic/ras-acc, I think this is fine for now. If it's alright, I'd like to address updating the email content if necessary in https://app.asana.com/0/1205234045751551/1205668937699542/f where we will be updating all transactional emails.~~ Scratch that, I found the copy for this here https://newspackdesignp2.wordpress.com/2023/12/05/cancellation-emails/. I will still be addressing the design updates in another pr, so this PR only gets the email setup with the old design.
 
### How to test the changes in this Pull Request:

1. Go to Newspack > Reader Revenue > Emails
2. Confirm a new Cancellation email is present as pictured above
3. Select the edit link
4. Confirm the email template post is populated with content relevant to donation cancellation
5. Make some changes to the post and update
6. Go back to the reader revenue emails menu
7. Select the reset link for the Cancellation email
8. Select the edit link once again and confirm your changes have been reset to default
9. As a reader, trigger a donation cancellation email by purchasing a recurring donation then cancelling via my account
10. Confirm you receive a cancellation email and its contents appear as expected (pictured above)
11. As a site admin, disable the cancellation email and repeat step 9
12. Confirm no cancellation email is sent

### Other information:
